src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.EasyRequest.cs

```diff
@@ -578,10 +591,17 @@ internal void  SetRedirectUri(Uri redirectUri)
                 _requestMessage.RequestUri = redirectUri;
             }
 
-            [Conditional(VerboseDebuggingConditional)]
-            private void VerboseTrace(string text = null, [CallerMemberName] string memberName = null)
+            private void EventSourceTrace<TArg0>(string formatMessage, TArg0 arg0, [CallerMemberName] string memberName = null)
```

Comment:
Am I right in assuming that  ```memberName``` parameter is set during the compile-time.  That is, somehow all the calls to this method are compiled to include an additional parameter (a constant string), added by the compiler?